src/os/ui/layer/compare/layercompareui.js

```diff
@@ -541,6 +547,45 @@ const launchLayerCompare = (options) => {
   }
 };
 
+
+/**
+ * Launch the layer compare.
+ * @param {!LayerCompareOptions} options The layer compare options.
+ */
+const launchLayerCompare = (options) => {
+  const featureCount = countFeatures(options.left) + countFeatures(options.right);
+  if (featureCount > getMaxFeatures('2d')) {
+    launchLayerComparePerformanceDialog().then(() => {
+      launchLayerCompareWindow(options);
+    }, () => {
+      // This empty block prevents an undefined error from appearing
```

Comment:
Very true, but doesn't explain "why". Promises will throw an error if they're rejected unless the rejection is handled, because rejection is by default considered to be an error. The "why" here is that we want to silently ignore the rejection because it was due to the user choosing not to open the window.